Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/GyrometerSamplePage.xaml.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.ComponentModel;
+using Uno.Gallery.ViewModels;
+using Windows.Devices.Sensors;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.NonUIFeatures, "Gyrometer",
+		Description = "Represents a Gyrometer sensor that provides angular velocity values with respect to the x, y, and z axes.",
+		DocumentationLink = "https://platform.uno/docs/articles/features/gyrometer.html",
+		DataType = typeof(GyrometerSamplePageViewModel))]
+	public sealed partial class GyrometerSamplePage : Page
+	{
+		public GyrometerSamplePage()
+		{
+			this.InitializeComponent();
+		}
+
+		private void ObserveReadingChangeButton_Click(object sender, RoutedEventArgs e)
+		{
+			if (!(((Sample)DataContext).Data is GyrometerSamplePageViewModel viewModel))
```

Comment:
```suggestion
			if (DataContext is not Sample { Data: GyrometerSamplePageViewModel vm })
```

Reply:
Right, but I am not sure if those work on UWP also. I think those are new features not available in UWP. I will double-check.

Reply:
<img width="490" alt="image" src="https://github.com/unoplatform/Uno.Gallery/assets/78549750/bdca4c29-7ef9-4b99-bcae-3693866c3325">